Providers/SelectedListManager.tscn

```diff
@@ -54,11 +61,10 @@ func reveal_hidden():
 	emit_signal(\"set_revealed_hidden\")
 	
 func remove_placemark(placemark, data):
-	toggle_selection(placemark, data)
-
-func close_details_panel(): # Closes the details panel if the list is empty
-	if selected_list.size() <= 0:
-		print(\"something is still calling this\")
+	var entry = {\"placemark\": placemark, \"data\": data}
+	var index = _find_entry_index(placemark)
```

Comment:
Kind of defeats the purpose if you have to find the index first. That should be done by the remove from selection function, it should take the placemark directly. That way it'd also be consistent with add_to_selection